graphene_django/tests/test_query.py

```diff
@@ -177,7 +187,7 @@ def resolve_event(self, info):
     assert result.data == expected
 
 
-def test_should_node():
+def test_should_node(use_dataloaders):
```

Comment:
I wonder is this fixture shouldn't better be named `execution_context_class` or similar, because the semantics of `execution_context_class=use_dataloaders` looks a little off.

Reply:
yeah that's a good idea, will update